docs-ref-autogen/azure-containerregistry/azure.containerregistry.ContainerRegistryClient.yml

```diff
@@ -32,7 +32,7 @@ constructor:
     description: 'URL to use for credential authentication with AAD. Its value could
       be
 
-      "[https://management.azure.com](https://management.azure.com)", "[https://management.chinacloudapi.cn](https://management.chinacloudapi.cn)"
+      "`https://management.azure.com`", "[https://management.chinacloudapi.cn](https://management.chinacloudapi.cn)"
```

Comment:
These proposals are drowning out any potentially interesting ones, can they be disabled and the tool rerun to see what the output looks like?

- These URLs are not actually links in the source, I believe the docs translator tool detects URLs and renders them as markdown links. For this small set of URLs that appears frequently, if the link-fixer is just going to remove them (and why is it only removing management.azure.com and not the others?), I think we'd be better off just not putting them in. Maybe we can detect and skip in the translator.
- Wrapping of URLs in backticks might be something we can do in the translator as well, at least for description fields (I don't think they get detected elsewhere, which is why they aren't Markdown links in fields like `syntax` and `defaultvalue`). The convention in the source appears to be to put them in double-quotes instead; having both together looks kind of funky and sort of defeats the point I think.

([Link](https://ceapex.visualstudio.com/Engineering/_git/py2docfx?path=/src/py2docfx/docfx_yaml/writer.py&version=GBdevelop&line=913&lineEnd=928&lineStartColumn=1&lineEndColumn=93&lineStyle=plain&_a=contents) to relevant translator source for posterity)